src/content/guidelines/content/glossary.mdx

```diff
@@ -1,7 +1,7 @@
 ---
 label: Content design can make or break an online experience. Always strive for writing that is clear, concise, and on-brand.
 title: Content
-tabs: ['General', 'Guidance', 'Glossary']
+tabs: ['Principles', 'Language & grammar', 'Capitalization', 'Punctuation', 'Numbers', 'Navigation & buttons', 'Glossary']
```

Comment:
All of these tabs aren't going to. fit FYI, is there another way we can organize this content?

Reply:
Oh, funny. I must have forgotten to remove them from that page. That was just experimentation, we aren't going with that structure. I'll remove them in my next push.